examples/csv_address_verification/verify.py

```diff
@@ -0,0 +1,65 @@
+# Usage
+# python verify.py input.csv
+
+import lob
+import csv
+import sys
+
+lob.api_key = 'test_0dc8d51e0acffcb1880e0f19c79b2f5b0cc'
+
+skipFirstLine  = True
+
+# columns in CSV
+address1    = 0;
+address2    = 1;
+city        = 2;
+state       = 3;
+postcode    = 4;
+country     = 'US'
+
+try:
+  sys.argv[1]
+except IndexError:
+  print "Please provide an input CSV file as an argument."
+  sys.exit()
+
+# Open Input File
```

Comment:
It's a minor thing but consider standardizing capitalization across the comments.